Frends.SFTP.ReadFile/Frends.SFTP.ReadFile.Tests/ReadTests.cs

```diff
@@ -5,88 +5,58 @@
 namespace Frends.SFTP.ReadFile.Tests;
```

Comment:
https://github.com/FrendsPlatform/Frends.SFTP/blob/c1ac6235cb127c66cb9e107ae893099c3b6a68cb/Frends.SFTP.ReadFile/Frends.SFTP.ReadFile.Tests/ReadTests.cs#L2
You are not using this namespace, so it can be removed.